test/unit/fixtures_adt_transaction.py

```
@@ -0,0 +1,97 @@
+<blue:blueSource xmlns:blue="http://www.sap.com/wbobj/blue" xmlns:adtcore="http://www.sap.com/adt/core" adtcore:type="TRAN/T" adtcore:description="abapgit" adtcore:language="EN" adtcore:name="{TRANSACTION_NAME}" adtcore:masterLanguage="EN" adtcore:responsible="DEVELOPER">
+<adtcore:packageRef adtcore:name="$TMP"/>
+</blue:blueSource>'''
+
+REPORT_TRANSACTION_CREATION_JSON = '{' \
+    '"abapLanguVersionText":"Standard ABAP",' \
+    '"transactionType":"reportTransaction",' \
+    '"reportName":"ZABAPGIT",' \
+    '"reportDynnr":"1000",' \
+    '"updateMode":"notSet",' \
+    '"metadata":{' \
+        '"name":"ZABAPGIT",' \
```

Flake8 (7.3.0)

[error] continuation line unaligned for hanging indent

(E131)